### ISSUE TITLE
MM is not working after latest update

### ISSUE THREAD

**Issue by gboeer** [ESOUIMods/MasterMerchant#47]
Since the latest Update (the Version updated by Minion) I cant use MM anymore. The Window doesn't open using /mm command and also the sales do not get parsed etc. It seems like it has something to do with LibGuildStore given the following error message I get after starting the game

```lua
user:/AddOns/MasterMerchant/Libs/LibGuildStore/Iterators/Iterators_Listings.lua:543: attempt to index a nil value
stack traceback:
user:/AddOns/MasterMerchant/Libs/LibGuildStore/Iterators/Iterators_Listings.lua:543: in function 'internal:ReferenceListings'
user:/AddOns/MasterMerchant/Libs/LibGuildStore/Iterators/Iterators_Listings.lua:497: in function 'internal:ReferenceListingsDataContainer'
user:/AddOns/MasterMerchant/Libs/LibGuildStore/LibGuildStore.lua:324: in function 'Queue'
user:/AddOns/MasterMerchant/Libs/LibExecutionQueue/LibExecutionQueue.lua:34: in function 'lib:executeNextTask'
user:/AddOns/MasterMerchant/Libs/LibExecutionQueue/LibExecutionQueue.lua:35: in function 'func'
/EsoUI/Libraries/Globals/globalapi.lua:261: in function '(anonymous)'
```

**Comment by gboeer** [comment]
With a little help I was able to solve this, by adding the following line:
```lua
local sales_data = _G["LibGuildStore_SalesData"]
```

at the top of the file:
   
    user:/AddOns/MasterMerchant/Libs/LibGuildStore/Iterators/Iterators_Listings.lua

**Comment by gboeer** [comment]
I think this was fixed with  beddb0cc31e4f86b698eb458154e9d0c062ab9b5
